packages/libosmocore/PKGBUILD

```diff
@@ -0,0 +1,38 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname=libosmocore
+pkgver=0.12.1
+pkgrel=1
+pkgdesc='Collection of common code used in various sub-projects inside the Osmocom family of projects.'
+arch=('x86_64' 'armv6h' 'armv7h' 'aarch64')
+groups=('blackarch' 'blackarch-radio')
+url='https://osmocom.org/projects/libosmocore/wiki/Libosmocore'
+license=('GPL')
+depends=('pcsclite')
```

Comment:
This is optional. Should be added to `optdepends` instead. If it's needed for the build process, add it to `makedepends` too.